module/VuFindTheme/tests/fixtures/vendor/example/res/theme/js/package.js

```diff
@@ -0,0 +1 @@
+/* file_get_contents */
```

Comment:
You seem to have new test fixtures, but no tests that use them. Am I overlooking something?